README.md

```diff
@@ -200,21 +200,23 @@ need different templates for different distributions.
 If some tasks need to be parameterized according to distribution and version (name of packages,
 configuration file paths, names of services), use this in the beginning of your `tasks/main.yml`:
 ```yaml
-- name: Set version specific variables
-  include_vars: "{{ item }}"
-  with_first_found:
-    - "{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml"
-    - "{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.yml"
-    - "{{ ansible_distribution }}.yml"
-    - "{{ ansible_os_family }}.yml"
+- name: Set distribution specific variables
+  include_vars: "{{ lookup('first_found', files) }}"
+  vars:
+    files:
+      - "{{ ansible_facts.distribution }}_{{ ansible_facts.distribution_version }}.yml"
+      - "{{ ansible_facts.distribution }}_{{ ansible_facts.distribution_major_version }}.yml"
+      - "{{ ansible_facts.os_family }}_{{ ansible_facts.distribution_major_version }}.yml"
+      - "{{ ansible_facts.distribution }}.yml"
+      - "{{ ansible_facts.os_family }}.yml"
 ```
 
+This list is organized from most specific to least specific. This allows variables common to an entire OS family or distribution to be specified in a single file, but also allows overriding a specific version that needs an exception. For example, variables common to most Fedora distributions could be placed in `Fedora.yml`. If a particular version needs different variables, create a file named `Fedora-32.yml`. This file would be found first and the more generic `Fedora.yml` settings would be ignored.
```

Comment:
This line is waaaaaaay too long

Reply:
Also, for some reason y'all chose to drop the "-" and go with "_" instead. So your example should read "`Fedora_32.yml`". That change broke with existing code and didn't seem to have a solid explanation of why it was made, but for consistency, the two should be made to line up.

Reply:
@greg-hellings "Also, for some reason y'all chose to drop the "-" and go with "_" instead." here: https://github.com/oasis-roles/meta_standards/pull/18

Reply:
I saw when, I just didn't see any discussion on **why** file names should follow the Python identifier naming convention. We have a mixed bag in OASIS at present.

```
$ find . -name '*-*.yml' | grep -v .tox
./system/roles/libvirtd/vars/RedHat-7.yml
./system/tests/create-docker.yml
./system/tests/create-openstack.yml
./system/tests/destroy-docker.yml
./system/tests/destroy-openstack.yml
./osp/tests/create-docker.yml
./osp/tests/create-openstack.yml
./osp/tests/destroy-docker.yml
./osp/tests/destroy-openstack.yml
./satellite/tests/create-docker.yml
./satellite/tests/create-openstack.yml
./satellite/tests/destroy-docker.yml
./satellite/tests/destroy-openstack.yml
```

Reply:
@robled you argued for file names following Python identifier naming conventions.

Reply:
I'll remove the explanation to make the line shorter, which will also get rid of the `-` vs `_` mistake.

For file names, I don't think it really matters that they be valid Python identifiers, but I don't care to debate that.